### PR TITLE
[MERGE] Forward port of saas-12.2 to saas-12.3 up to cf8e74a1a7d34

### PR DESCRIPTION
**Pull request by mart-e**

### CODE REVIEW

**Comment by mart-e** [comment]
@aab-odoo cf the "do not forward port part" on 8f9e1e40a32748e576084320

@kebeclibre tried to adapt c8d10d228e722e3476e845c58e96

@Julien00859 tried to adapt b2b639b693dd7215e3ee501ac89c35f3a6b88bb5

Thanks for your help

File: addons/web/static/tests/fields/relational_fields_tests.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ var relationalFields = require('web.relational_fields');
 var testUtils = require('web.test_utils');
 
 var createView = testUtils.createView;
+var createAsyncView = testUtils.createAsyncView;
```
Comment (aab-odoo) [inline addons/web/static/tests/fields/relational_fields_tests.js]: remove

File: addons/web/static/src/js/views/basic/basic_model.js [hunk 2]
```diff
@@ -159,6 +159,7 @@ var BasicModel = AbstractModel.extend({
         // stack, so that they can be batched in the minimum number of RPCs
         this.batchedRPCsRequests = [];
 
+
```
Comment (aab-odoo) [inline addons/web/static/src/js/views/basic/basic_model.js]: remove

**Comment by mart-e** [comment]
@robodoo merge review+ priority=0

**Comment by mart-e** [comment]
@robodoo retry

**Pull request closed without merging**